[4.0] Änderung der Standardsprache

tecpromotion opened the pull request:
Pull Request für Issue #1998 .

### Zusammenfassung der Änderungen

Optimierter Sprachstring

### Wo wird der Sprachstring angezeigt / Wie kann getestet werden

both

tecpromotion committed:
fix #1998

chmst commented:
Das finde ich schon gut. 

Mankönnte das noch kürzen:
Bei mehrsprachigen Websites (wenn das „System - Sprachfilter“-Plugin aktiviert ist) ist es zwingend, dass die Standardsprache der Website ebenfalls eine veröffentlichte Inhaltssprache ist

zu 

Bei mehrsprachigen Websites (wenn das „System - Sprachfilter“-Plugin aktiviert ist) muss die Standardsprache der Website eine veröffentlichte Inhaltssprache sein.

tecpromotion committed:
fix #1998

Thanks @chmst
Habs noch kürzer geschrieben und das zweite Website weggelassen, sollte so auch klar sein.
